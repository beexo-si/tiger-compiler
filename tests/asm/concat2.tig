let
    var s := concat(
		"a", "")
in
    print(s);
    0
end
