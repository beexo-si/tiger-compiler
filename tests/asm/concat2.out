bien!
yes!!
aReturn code: 0
